File: src/pocketcloud/event/impl/server/ServerStopEvent.php
<?php

namespace pocketcloud\event\impl\server;

use pocketcloud\server\CloudServer;

class ServerStopEvent extends ServerEvent {

    public function __construct(
        CloudServer $server,
        private readonly bool $force
    ) {
        parent::__construct($server);
    }

    public function isForce(): bool {
        return $this->force;
    }
}

File: cloud-core/src/main/java/de/pocketcloud/cloud/event/impl/server/ServerStopEvent.java
package de.pocketcloud.cloud.event.impl.server;

import de.pocketcloud.cloud.server.CloudServer;
import lombok.Getter;

@Getter
public class ServerStopEvent extends ServerEvent {

    private final boolean forcefully;

    public ServerStopEvent(CloudServer server, boolean forcefully) {
        super(server);
        this.forcefully = forcefully;
    }
}